ocs_ci/deployment/deployment.py

```diff
@@ -199,6 +200,102 @@ def do_deploy_submariner(self):
             submariner = Submariner()
             submariner.deploy()
 
+    def do_gitops_deploy(self):
+        """
+        Deploy GitOps operator
+
+        Returns:
+
+        """
+        if not config.ENV_DATA.get("deploy_gitops_operator"):
+            return
+
+        # Multicluster operations
+        if config.multicluster:
+            config.switch_acm_ctx()
+            logger.info("Creating GitOps Operator Subscription")
```

Comment:
What does Subscription mean here?

Reply:
Subscription is ocp operator subscription